### DISCUSSION TITLE
Is there a way to add `LocaleSwitcher` to a `RelationManager` form modal?

### DISCUSSION THREAD

**Discussion by nabileisa** [filamentphp/filament#9576] [Help]
Hi , 
Is there a way to add `LocaleSwitcher` to a `RelationManager` form modal?
I would like to be able to add  or modify a relationship with multiple  locales - in a modal -  without having to save and switch the locale and reopen the modal again. Is that possible?

Thanks

**Comment by leandrocfe** [comment]
Would you like to customize the form? There is an example [here](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#attaching-with-pivot-attributes)

**Reply by nabileisa** [reply]
No actually, I'm trying to add the `LocaleSwitcher` action somewhere inside the relation manager modal preferably in the header.
My goal here is that I can switch between multiple locales while the modal is open, and be able to add or modify content  in multiple locales at once then save, as oppose to the current flow where I have to add one language then save then switch the locale then edit the relationship record to add the other translation.

**Reply by broqit** [reply]
Hello. Have you finally solved this case? I have the same problem.

**Reply by nabileisa** [reply]
@broqit I ended up realizing that  The local switcher is not a very good solution and did not support this use case .
My suggestion would be to use an translation tabs plugin that allow you to group your translatable  fields in a tab per language , 
like this one
 https://github.com/solutionforest/filament-translate-field
or this one which already support the v4 beta
 https://filamentphp.com/plugins/abdulmajeed-jamaan-translatable-tabs
 They can be used anywhere that a form is displayed and more convenient than the local switcher.
 Hope this helps.

**Comment by thomas-alrek** [comment]
+1, is this possible? I want to be able to switch locale inside the form of the RelationManager

**Reply by broqit** [reply]
Did you find solution?
